Cargo.toml

```diff
@@ -31,7 +31,7 @@ futures = "0.1"
 futures-cpupool = "0.1"
 fdlimit = "0.1"
 ctrlc = { git = "https://github.com/paritytech/rust-ctrlc.git" }
-jsonrpc-core = { git = "https://github.com/paritytech/jsonrpc.git", branch = "parity-1.11" }
+jsonrpc-core = { git = "https://github.com/c0gent/jsonrpc.git", branch = "c0gent-hyper" }
```

Comment:
This will change back to `"https://github.com/paritytech/jsonrpc.git"` at some point before the PR lands right?

Reply:
Absolutely. Currently just waiting on a specific release/branch/tag etc. to depend on (It's probably wise to hold off on that until this PR is confirmed to work properly).